src/components/discovery/VariantSearchHeader.js

```diff
@@ -2,8 +2,12 @@ import React, { useState } from "react";
 import { Form, Select } from "antd";
 import PropTypes from "prop-types";
 import LocusSearch from "./LocusSearch";
+import { useSelector } from "react-redux";
 
 const VariantSearchHeader = ({dataType, addVariantSearchValues}) => {
+    const varOvRes = useSelector((state) => state.explorer.variantsOverviewResponse);
+    const ovAsmIds = varOvRes?.assemblyIDs !== undefined ? Object.keys(varOvRes?.assemblyIDs) : [];
```

Comment:
variable names here are obscure to me

Reply:
I agree
The linter doesn't like lines longer than 120 characters, so I abbreviated them

Reply:
see https://github.com/bento-platform/bento_web/pull/199/commits/d70960701bfb1b8a67ef59bbfe7d0c3f336471cc for the nice-to-have!